examples/misc/lib/pi_monte_carlo.dart

```diff
@@ -20,6 +22,7 @@ int numIterations = 500; //!web-only
 // after 500 iterations). //!web-only
 //!web-only
 //!tip("main()")
+// #docregion try-dart
 main() async {
```

Comment:
Rather than create another "Compute Pi" that was console-only, I'm reusing the HTML version, and stripping out the HTML-specific code. (This version of the code actually had updated syntax as compared to the original try-dart code.)

Reply:
Code change undone.